pkg/extensions/sync/sync_test.go

```diff
@@ -2227,7 +2459,7 @@ func TestSyncSignaturesOnDemand(t *testing.T) {
 		// sync on demand
 		resp, err = resty.R().Get(destBaseURL + "/v2/" + repoName + "/manifests/1.0")
 		So(err, ShouldBeNil)
-		So(resp.StatusCode(), ShouldEqual, 404)
+		So(resp.StatusCode(), ShouldEqual, 200)
```

Comment:
Had to change it, sync signatures won't return error anymore, because the image was successfully synced already and next time we will request it it will work, but the signature won't be there, so either the signature is not on the upstream or there was a temporary error and we will get it eventually.